docs/io/fits/usage/headers.rst

```diff
@@ -380,18 +425,21 @@ Examples follow::
     EXTEND  =                    T
     HIERARCH abcdefghi = 10
 
+..
+  EXAMPLE END
+
 .. note::
 
     A final point to keep in mind about the :class:`Header` class is that much
     of its design is intended to abstract away quirks about the FITS format.
-    This is why, for example, it will automatically created CONTINUE and
-    HIERARCH cards.  The Header is just a data structure, and as user you
-    shouldn't have to worry about how it ultimately gets serialized to a header
+    This is why, for example, it will automatically create CONTINUE and
+    HIERARCH cards. The Header is just a data structure, and as a user you
+    should not have to worry about how it ultimately gets serialized to a header
     in a FITS file.
 
-    Though there are some areas where it's almost impossible to hide away the
-    quirks of the FITS format, Astropy tries to make it so that you have to
-    think about it as little as possible.  If there are any areas where you
-    have concern yourself unnecessarily about how the header is constructed,
-    then let help@stsci.edu know, as there are probably areas where this can be
+    Though there are some areas where it is almost impossible to hide away the
+    quirks of the FITS format, ``astropy`` tries to make it so that you have to
+    think about it as little as possible. If there are any areas where you
```

Comment:
This sentence is phrased strangely "...where you have concerned yourself unnecessarily..." - maybe you can rephrase this?